x-pack/plugins/enterprise_search/public/applications/shared/schema/add_field_modal/index.tsx

```diff
@@ -78,65 +79,64 @@ export const SchemaAddFieldModal: React.FC<ISchemaAddFieldModalProps> = ({
+                  placeholder={FIELD_NAME_PLACEHOLDER}
+                  type="text"
+                  onChange={handleChange}
+                  required
+                  value={rawFieldName}
                   fullWidth
-                  data-test-subj="SchemaAddFieldNameRow"
-                  error={addFieldFormErrors}
-                  isInvalid={!!addFieldFormErrors}
-                >
-                  <EuiFieldText
-                    placeholder={FIELD_NAME_PLACEHOLDER}
-                    type="text"
-                    onChange={handleChange}
-                    required
-                    value={rawFieldName}
-                    fullWidth
-                    autoFocus
-                    isLoading={loading}
-                    data-test-subj="SchemaAddFieldNameField"
-                  />
-                </EuiFormRow>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiFormRow label={FIELD_TYPE} data-test-subj="SchemaAddFieldTypeRow">
-                  <SchemaFieldTypeSelect
-                    fieldName=""
-                    fieldType={newFieldType}
-                    updateExistingFieldType={(_, type: SchemaType) => updateNewFieldType(type)}
-                    disabled={disableForm}
-                    data-test-subj="SchemaSelect"
-                  />
-                </EuiFormRow>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiForm>
-        </EuiModalBody>
-        <EuiModalFooter>
-          <EuiButtonEmpty onClick={closeAddFieldModal}>{CANCEL_BUTTON_LABEL}</EuiButtonEmpty>
-          <EuiButton
-            color="primary"
-            fill
-            disabled={disableForm}
-            type="submit"
-            isLoading={loading}
-            data-test-subj="SchemaAddFieldAddFieldButton"
-          >
-            {ADD_FIELD_BUTTON}
-          </EuiButton>
-        </EuiModalFooter>
-      </form>
+                  autoFocus
+                  isLoading={loading}
+                  data-test-subj="SchemaAddFieldNameField"
+                />
+              </EuiFormRow>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiFormRow label={FIELD_TYPE} data-test-subj="SchemaAddFieldTypeRow">
+                <SchemaFieldTypeSelect
+                  fieldName=""
+                  fieldType={newFieldType}
+                  updateExistingFieldType={(_, type: SchemaType) => updateNewFieldType(type)}
+                  disabled={disableForm}
+                  data-test-subj="SchemaSelect"
+                />
+              </EuiFormRow>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiForm>
+      </EuiModalBody>
+      <EuiModalFooter>
+        <EuiButtonEmpty onClick={closeAddFieldModal}>{CANCEL_BUTTON_LABEL}</EuiButtonEmpty>
+        <EuiButton
+          color="primary"
+          fill
+          disabled={disableForm}
+          type="submit"
+          form={FORM_ID}
```

Comment:
TIL

Reply:
Ya same!! Only just learnt this last week while creating the synonyms modal lol.